examples/ios/objc/Puzzle/Views/RLMPuzzleView.m

```diff
@@ -0,0 +1,212 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+#import "RLMPuzzleView.h"
+#import "RLMPuzzlePieceView.h"
+
+@interface RLMPuzzleView ()
+
+@property (nonatomic, strong) NSArray *puzzlePieces;
+@property (nonatomic, strong) NSString *pieceNames; //String versions of the piece names
+@property (nonatomic, strong) NSDictionary *pieceDefaultLocations; //The CGPoints for each piece in default space
+@property (nonatomic, assign) CGPoint gestureOrigin;
+@property (nonatomic, assign) CGPoint gesturePieceOrigin;
+@property (nonatomic, weak)   UIView *draggingView;
+
+- (void)setupPuzzlePieces;
```

Comment:
same with these declarations, I don't think they're used from outside the file, so why forward-declare them here?